src/orderbook-basics.yaml

```diff
@@ -12,23 +12,371 @@ publishStatus: Live
+      - uuid: 984d2c1a-b521-433f-98ca-3a84035942bb
+        content: Which of the following DEXs use CLOB?
+        answerKeys:
+          - A
+          - B
+          - C
+        type: MultipleChoice
+        choices:
+          - content: Serum
+            key: A
+          - content: Coinbase
+            key: B
+          - content: Kraken
+            key: C
+          - content: TerraSwap
+            key: D
+
+
+  - uuid: b665e2cb-dd10-45c4-8de0-bf1595eb18aa
+    name: Serum-Deep Dive
+    content: |
+      ### Introduction to Serum
+      Built on the Solana network, Serum is a very effective decentralised exchange that is also quite scalable and therefore widely accepted. Serum's platform uses Central Limit Order Book (CLOB) rather than Automated Market Makers (used by others) to facilitate efficient and quick trading, which is the key distinction between Serum and other well-known DEXs. 
+      
+      Serum's built-in functionality is enhanced by its on-chain CLOB, a commonly used mechanism in traditional finance. Additionally, Serum makes it permissionless and decentralised, enabling users to communicate with a smart contract directly to execute trades from an order book without the need for a middleman.
+
+      ### Central Limit Order Book 
+      The CLOB model, which enables users to clearly match their other with others on a "price-time-priority" basis, is one of Serum's most helpful design elements. Customers could opt to cross this bid-ask spread and have their order promptly filled in this model's order book where the market price is represented by the highest bid and lowest ask.
+
+      High levels of transparency offered by the CLOB model allow customers to monitor market depth in real time. Although CLOB has very powerful features, integrating it into an on-chain system is exceedingly difficult. The key reasons for this difficulty are order books' high throughput and cheap execution costs. Therefore, low gas costs and rapid transactions are necessary for a CLOB to operate on-chain.
+
+      Many of the first DEX protocols were conscious of these challenges and made an effort to replace the traditional CLOB paradigm with an AMM. By employing their AMM technology, Uniswap's Defi users were able to complete transactions rapidly and affordably.    
     stepItems: []
-  - uuid: 7994939b-5188-4951-aaf1-e3a268725fb6
+
+  - uuid: 4dfc846e-09b4-4c51-8374-dfd1e3e5048e
+    name: Evaluation
+    content: |
+
+    stepItems:
+      - uuid: c7b5cdea-2928-4dd7-8da2-d786fd98187f
+        content: Serum uses which of the following builders?
+        answerKeys:
+          - C
+        type: SingleChoice
+        choices:
+          - content: Ethereum
+            key: A
+          - content: Hyperledger
+            key: B
+          - content: Solana
+            key: C
+          - content: R3
+            key: D
+      - uuid: 44a8a64e-2fb5-43c9-9392-36a61334831d
+        content: Serum CLOB works on the basis of-
```

Comment:
CLOB uses both price and time so its not purely FIFO

Reply:
Also use "CLOB" -> "Center Limit Order Book"

Reply:
Made it more intuitive